Doc/whatsnew/3.14.rst

```diff
@@ -90,6 +90,29 @@ If you encounter :exc:`NameError`\s or pickling errors coming out of
 New features
 ============
 
+PEP 758 – Allow except and except* expressions without parentheses
+------------------------------------------------------------------
+
+The :keyword:`except` and :keyword:`except*` expressions now allow
```

Comment:
```suggestion
The :keyword:`except` and ``except*`` <except_star>` expressions now allow
```

`except*` is not a keyword. This will cause the docs build and workflow to fail. We can using ``except*``
 https://github.com/python/cpython/blob/main/Doc%2Ftutorial%2Ferrors.rst#L557-L557

Reply:
This was suggested here: https://github.com/python/cpython/pull/131833#discussion_r2019158994